flask_github.py

```diff
@@ -81,6 +120,31 @@ def authorize(self, scope=None, redirect_uri=None):
         """
         Redirect to GitHub and request access to a user's data.
 
+        :param scope: List of `Scopes`_ for which to request access, formatted
+                      as a string or comma delimited list of scopes as a
+                      string. Defaults to ``None``, resulting in granting
+                      read-only access to public information (includes public
+                      user profile info, public repository info, and gists).
+        :type scope: str
+        :param redirect_uri: `Redirect URL`_ to which to redirect the user
```

Comment:
That's true, GitHub will raise an error if the `redirect_uri` doesn't "match" the one configured in the application, and, [if omitted from a request, it just uses the application defined URL anyway](https://developer.github.com/v3/oauth/#redirect-urls).

There is an exception though, that so long as the `redirect_uri` is a subdirectory of the one configured in the application, it should be okay.

> The redirect URL’s path must reference a subdirectory of the callback URL.